nacc/uds3/ivp/builder.py

```diff
@@ -155,323 +155,338 @@ def add_z1_or_z1x(record, packet):
         packet.insert(0, z1)
 
 
-def add_a1(record,packet):
```

Comment:
Thank you for adhering to the "no flake8 warnings" rule; this is, however, another instance of where I think it's safe to ignore the warnings as the code is more readable with the aligned `=`'s.